tests/integration/test_fim/test_registry/test_registry_wildcards/test_registry_wildcards.py

```diff
@@ -0,0 +1,301 @@
+t1_configurations = load_configuration_template(configurations_path, t1_configuration_parameters,
+                                                t1_configuration_metadata)
+
+t2_configuration_parameters, t2_configuration_metadata, t2_case_ids = get_test_cases_data(t2_cases_path)
+t2_configurations = load_configuration_template(configurations_path, t2_configuration_parameters,
+                                                t2_configuration_metadata)
+
+# Variables
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+key_name = 'test_key'
+value_name = 'test_value'
+
+
+# Tests
+@pytest.mark.parametrize('configuration, metadata', zip(t1_configurations, t1_configuration_metadata), ids=t1_case_ids)
+def test_registry_key_wildcards(configuration, metadata, set_wazuh_configuration, truncate_monitored_files,
+                                configure_local_internal_options_function, restart_wazuh_function,
+                                wait_syscheck_start):
+    '''
+    description: Check the behavior of FIM when using wildcards to configure the path of registry keys, and validate
+                 the keys creation, modification and deletion is detected correctly.
+
+    wazuh_min_version: 4.5.0
+
+    test_phases:
+        - setup:
+            - Set wazuh configuration.
+            - Clean logs files and restart wazuh to apply the configuration.
+        - test:
+            - Check that one or more keys are detected when the configured wildcard is expanded
+            - Create a subkey inside the first monitored key and check
+            - Wait for scan and check subkey has been detected as 'added'
+            - Modify the subkey
+            - Wait for scan and check subkey has been detected as 'modified'
+            - Delete the subkey
+            - Wait for scan and check subkey has been detected as 'deleted'
+        - teardown:
+            - Restore configuration
+            - Stop wazuh
+
+    tier: 1
+
+    parameters:
+        - configuration:
+            type: dict
+            brief: Configuration values for ossec.conf.
+        - metadata:
+            type: dict
+            brief: Test case data.
+        - set_wazuh_configuration:
+            type: fixture
+            brief: Set ossec.conf configuration.
+        - configure_local_internal_options_function:
+            type: fixture
+            brief: Set local_internal_options configuration.
+        - policies_file:
+            type: string
+            brief: path for audit policies file to use on restore_win_whodata_policies fixture
+        - restore_win_whodata_policies
+            type: fixture
+            brief: restores windows audit policies using a given csv file after yield
+        - restart_syscheck_function:
+            type: fixture
+            brief: restart syscheckd daemon, and truncate the ossec.log.
+        - wait_for_fim_start_function:
+            type: fixture
+            brief: check that the starting fim scan is detected.
```

Comment:
Please, add the correct parameters.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4220/commits/cc756e243308370882f0aab65b88291a92535154